tensorflow/core/kernels/mkl_fused_ops_test.cc

```diff
@@ -295,11 +393,38 @@ TYPED_TEST_P(MklFusedConv2DWithBiasOpTest, SpatialConvolutionAndRelu) {
   this->VerifyConv2DWithBiasAndRelu(filter_size, filter_count);
 }
 
+TYPED_TEST_P(MklFusedConv2DWithBiasOpTest, OneByOneConvolutionAndAdd) {
+  const int filter_size = 1;
+  const int filter_count = 12;
+  this->VerifyConv2DWithBiasAndAdd(filter_size, filter_count, filter_count);
+}
+
+TYPED_TEST_P(MklFusedConv2DWithBiasOpTest, SpatialConvolutionAndAdd) {
+  const int filter_size = 3;
+  const int filter_count = 12;
+  this->VerifyConv2DWithBiasAndAdd(filter_size, filter_count, filter_count);
+}
+
+TYPED_TEST_P(MklFusedConv2DWithBiasOpTest, OneByOneConvolutionAndAddRelu) {
+  const int filter_size = 1;
+  const int filter_count = 12;
+  this->VerifyConv2DWithBiasAndAddRelu(filter_size, filter_count, filter_count);
+}
+
+TYPED_TEST_P(MklFusedConv2DWithBiasOpTest, SpatialConvolutionAndAddRelu) {
+  const int filter_size = 3;
+  const int filter_count = 12;
+  this->VerifyConv2DWithBiasAndAddRelu(filter_size, filter_count, filter_count);
+}
+
 REGISTER_TYPED_TEST_CASE_P(MklFusedConv2DWithBiasOpTest,  //
                            OneByOneConvolution,           //
                            SpatialConvolution,            //
                            OneByOneConvolutionAndRelu,    //
-                           SpatialConvolutionAndRelu);
+                           SpatialConvolutionAndRelu, OneByOneConvolutionAndAdd,
```

Comment:
I think one line for each case looks better. If you add `//` at the end of each line, the formatter will leave them like that.